tests/python/multidevice/linear.py

```diff
@@ -85,11 +88,12 @@ def forward(
         weight: DTensor,
     ):
         assert input.dim() in (2, 3)
+        in_dtensors = [input, weight]
```

Comment:
I'd revert the change and make in_dtensors inline. They are used once after all.